pipeline/dbt/seeds/schema/profils.csv

```diff
@@ -24,4 +25,5 @@ sans-domicile-fixe,Sans domicile fixe,
 seniors-65,Seniors (+ 65 ans),
 sortants-de-detention,Sortants de détention,
 surdite,Surdité,
+tous-publics,Tout publics,
```

Comment:
dans models/intermediate/sources/soliguide/int_soliguide__services.sql on a une référence à `tout-publics` et pas `tous-publics`, which is good ?

Reply:
good catch :+1: